### PR TITLE
[Bugfix]Fixed showing events

### PR DESCRIPTION
**Pull request by mhib**

### CODE REVIEW

File: app/models/google_event.rb [hunk 1]
```diff
@@ -89,6 +88,11 @@ def load_emails
       ConferenceRoom.pluck(:email)
     end
 
+    def normalize_dates(event)
+      event.start.date_time = new_time_low event.start.date_time
```
Comment (pbzymek) [inline app/models/google_event.rb]: missing ()

``` ruby
event.start.date_time = new_time_low(event.start.date_time)
```

File: app/models/google_event.rb [hunk 2]
```diff
@@ -89,6 +88,11 @@ def load_emails
...
+      event.end.date_time = new_time_high event.end.date_time
+    end
+
     GRANULARITY = 30.minutes.freeze
     def new_time_low(time)
```
Comment (pbzymek) [inline app/models/google_event.rb]: could you think of a more descriptive method name ?

File: app/models/google_event.rb [hunk 3]
```diff
@@ -100,7 +104,7 @@ def new_time_low(time)
 
     def new_time_high(time)
```
Comment (pbzymek) [inline app/models/google_event.rb]: could you think of a more descriptive method name ?

Comment (mhib) [inline app/models/google_event.rb]: floor_time and ceil_time would be ok?

Comment (pbzymek) [inline app/models/google_event.rb]: is this something use only on the view?

Comment (mhib) [inline app/models/google_event.rb]: It is required by the grouping algorithm.

File: app/models/google_event.rb [hunk 4]
```diff
@@ -100,7 +104,7 @@ def new_time_low(time)
...
       if time > time.beginning_of_hour + GRANULARITY
-        time.beginning_of_hour + 2 * GRANULARITY
+        time.beginning_of_hour + GRANULARITY + GRANULARITY
```
Comment (madejejej) [inline app/models/google_event.rb:130]: Why this change?

Comment (mhib) [inline app/models/google_event.rb:130]: Because GRANULARITY was transformed into integer and instead of 1 hour, 10 years were added.
